index.html

```diff
@@ -1,14 +1,27 @@
 <!DOCTYPE html>
 <html lang="en">
 <head>
-	<meta charset="utf-8"/>
-	<meta http-equiv="x-ua-compatible" content="ie=edge"/>
-	<meta name="viewport" content="width=device-width, initial-scale=1"/>
-	<title>Surfing</title>
-	<link rel="stylesheet" type="text/css" href="normalize.css">
-	<link rel="stylesheet" type="text/css" href="index.css">
+    <meta charset="utf-8"/>
+    <meta http-equiv="x-ua-compatible" content="ie=edge"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1"/>
+    <title>Surfing</title>
+    <link rel="stylesheet" type="text/css" href="normalize.css" />
+    <link rel="stylesheet" type="text/css" href="index.css" />
 </head>
 <body>
+<div class="wrapper">
+     <header class="header">
+        <nav class="header__nav">
+            <img class="header__logo" src="images/logo.svg" alt="header logo" width="16px" height="44px" />
+            <ul class="list">
+                <li class="list__item"><a class="list__link font-primary" href="#">stories</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">events</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">places</a></li>
+                <li class="list__item"><a class="list__link font-primary" href="#">boards</a></li>
```

Comment:
I think you need to use the `font-primary` class on the parent, only once, no need to use it on every single `<a>` tag.